.eslintrc.cjs

```diff
@@ -1,16 +1,21 @@
 module.exports = {
+  root: true,
   env: {
     es2021: true,
     node: true,
   },
-  extends: ['eslint:recommended', 'plugin:n/recommended'],
+  extends: ['eslint:recommended', 'plugin:n/recommended', 'plugin:@typescript-eslint/recommended'],
```

Comment:
We're extending only recommended set of typescript rules here. Additionally, we can include [requiring-type-checking](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/recommended-requiring-type-checking.ts) set of rules. It will add recommended rules that use type information with the caveat that it will take longer to run lint process.

I tried it locally and it gave me about 70 more issues (many `any`-related and async/promises errors). 

Any thoughts on adding it?